lib/src/flexible_scrollbar.dart

```diff
@@ -344,6 +344,14 @@ class _FlexibleScrollbarState extends State<FlexibleScrollbar> {
                         !isVertical && widget.draggable ? onDragUpdate : null,
                     onHorizontalDragEnd:
                         !isVertical && widget.draggable ? onDragEnd : null,
+                    onPanDown: (details) {
```

Comment:
fixing the unexpected scrollbar thumb's behavior when panning. it's offset incremented twice as fast as it should have